src/main/java/net/revelc/apilyzer/maven/plugin/Analyze.java

```diff
@@ -123,25 +117,17 @@ private boolean checkClass(Class<?> clazz, Set<String> publicSet, PrintStream ou
       return true;
     }
 
-    Class<?> superClazz = clazz.getSuperclass();
-    if (superClazz != null) {
-      ok = checkSuperClass(clazz, publicSet, out, ok, superClazz);
-    }
-
-    for (Class<?> iface : clazz.getInterfaces()) {
-      ok = checkSuperClass(clazz, publicSet, out, ok, iface);
-    }
-
     // TODO check generic type parameters
 
-    Field[] fields = clazz.getDeclaredFields();
+    Field[] fields = clazz.getFields();
```

Comment:
Damn it, I thought this method would return those.  It does not.  Not seeing an easy way to get inherited protected methods.  Would have to recurse up. When recursing up, would need to dedupe taking 1st (handle case where subclass overrides superclass protected method and changes return type).